build-tools/packages/build-cli/package.json

```diff
@@ -89,6 +89,7 @@
 		"inquirer-table-prompt": "^0.2.1",
 		"jssm": "^5.79.18",
 		"jssm-viz-cli": "^5.83.0",
+		"minimatch": "^3.0.4",
```

Comment:
This version is 6 years old, has a CVE ( https://nvd.nist.gov/vuln/detail/CVE-2022-3517 ) and is 4 major versions out of date. Also the newer version have typescript types built in. Could we use something newer?

Reply:
Yeah -- it pulled this version since it was used elsewhere. I'll update them both.

Reply:
Actually I don't know why this version was used. Regardless, updated.